docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,392 @@
+
+
+### What is a package?
+There are several ways to deploy your service onto a running DC/OS cluster.
+  * Use the DC/OS Marathon command in the CLI,
+  * Use the Marathon REST API directly, or
+  * Deploy your service as a package.
+
+Deploying your service using the package approach makes your life easier and service management efficient. Once you have a running DC/OS cluster, you will be able to browse packages in the dashboard. A package consists of the four required configuration files(`config.json`, `package.json`, `resource.json`, and `marathon.json.mustache`) and all of the external files linked from them.
+
+A package implicitly relies on Marathon; its contents are used to generate a Marathon app definition. By the end of this guide, you will be able to build, publish, and browse your package in the cluster.
+
+
+## Prerequisites
+Before starting this guide, make sure you have met the following conditions.
+
+### Library dependencies
+* [DC/OS CLI](https://dcos.io/docs/latest/cli/install/) installed and configured.
+* [jq](https://stedolan.github.io/jq/download/) is installed in your environment.
+* `python3` in your environment.
+* Docker is installed.
+
+### Access requirements
+* Access to a running [DC/OS](https://dcos.io/install/).
+* The Universe Server needs to be built and run in a location accessible by the DC/OS Cluster. There is no other way to test your package otherwise.
+* Mesos needs access to the Docker registry that has your Universe Server. In our guide, we use Docker Hub as the registry.
+
+
+### This repository
+- Current packaging version is v4 and we follow this standard for this guide. This guide will be updated as and when we release a new version.
+- The packages are located in `repo/packages` directory.
+- This tutorial is in `tutorial` directory
+- You can refer to **schemas** in `repo/meta/schema` directory. This directory has
+  - `config-schema.json` that refers to the schema of the `config.json`
+  - `package-schema.json` that refers to the schema of the `package.json`
+  - `v3-resource-schema.json` that refers to the schema of the `resource.json` for the v3 and v4 packages
+  - `*-repo-schema.json` files are not meant to be used by package developers; they instead define the schema for the API between Universe and DC/OS.
+
+## Create a package
+Let us build a simple Python HTTP server, which, when it receives an HTTP GET request, responds with the current time at the server. We will start with this and build a package that provides this Python server as a DC/OS service.
+
+
+### Step 1 : Create a simple Python HTTP Server
+For the purposes of this guide, we will be using Python 3 and the HTTPServer module provided by its standard library. Let's create a file called `helloworld.py` in an empty directory called `time-server-service`
```

Comment:
suggest: `For the purposes of this guide, Python 3 and the HTTPServer module provided by its standard library is used.`